### PR TITLE
Added tests and miscellaneous changes

### PR DESCRIPTION
**Pull request by MikeTschudi**
Restored coverage testing

### CODE REVIEW

File: packages/simple-types/test/dashboard.test.ts [hunk 1]
```diff
@@ -125,6 +126,36 @@ describe("Module `dashboard`: manages the creation and deployment of dashboard i
...
+            type: "serviceDataset"
+          }
+        ]
+      };
+
+      const itemTemplate: common.IItemTemplate = templates.getItemTemplate(
+        "Dashboard"
+      );
+
+      dashboard._getDatasourceDependencies(obj, itemTemplate);
```
Comment (jmhauck) [inline packages/simple-types/test/dashboard.test.ts]: @MikeTschudi should these have some sort of expect at the end or ok with the fact that they simply complete?

File: packages/simple-types/test/dashboard.test.ts [hunk 2]
```diff
@@ -194,4 +225,76 @@ describe("Module `dashboard`: manages the creation and deployment of dashboard i
...
   });
+
+  describe("_templatizeByDatasource", () => {
+    it("ignores supplied objs if it is not defined", () => {
+      const updatedList = dashboard._templatizeByDatasource(null, null);
+      expect(updatedList).toBeNull();
+    });
+  });
+
+  describe("_getDatasourceInfo", () => {
```
Comment (jmhauck) [inline packages/simple-types/test/dashboard.test.ts]: same here?

File: packages/common/src/interfaces.ts [hunk 3]
```diff
@@ -360,21 +360,6 @@ export interface IItemTemplate {
   estimatedDeploymentCostFactor: number;
 }
 
-export interface ICreateItemFromTemplateResponse {
```
Comment (jmhauck) [inline packages/common/src/interfaces.ts]: @MikeTschudi should this have been added back somewhere?

Comment (jmhauck) [reply packages/common/src/interfaces.ts]: oh wait...I see...when I look back at the source...it was a second instance of this

File: packages/simple-types/test/dashboard.test.ts (hunk 2 above)
Comment (MikeTschudi) [reply packages/simple-types/test/dashboard.test.ts]: @jmhauck, these look like merge errors that I made; I'll fix; thanks

**Pull request merged**